provider/pkg/kinds/kinds.go

```diff
@@ -196,3 +202,15 @@ const (
 	StorageV1A1               groupVersion = "storage.k8s.io/v1alpha1"
 	StorageV1B1               groupVersion = "storage.k8s.io/v1beta1"
 )
+
+// toGVK is a helper function that converts the internal groupVersion and Kind types to a schema.GroupVersionKind
+func toGVK(gv groupVersion, kind Kind) schema.GroupVersionKind {
```

Comment:
I could be mistaken, but doesn't a similar helper function exist somewhere?

Reply:
There are similar methods that take string inputs, but not these internal types.